src/DynamoCore/Graph/Nodes/NodeModel.cs

```diff
@@ -503,36 +503,81 @@ public string DictionaryLink
         {
             get
             {
-                string finalLink = Configurations.DynamoDictionary + "#/";
-                if (category == null || category == "")
-                {
-                    return Configurations.DynamoDictionary; // if there is no category, return the link to home page
-                }
+                dictionaryLink = dictionaryLink ?? Configurations.DynamoDictionary;
+                return dictionaryLink;
+            }
+            set
+            {
+                dictionaryLink = value;
+            }
+        }
+
+        private string dictionaryLink;
+
+        public string GetDictionaryLinkFromLibrary(LibraryServices libraryServices)
+        {
+            string finalLink = Configurations.DynamoDictionary + "#/";
+            if (IsCustomFunction)
+            {
+                return ""; // If it is not a core or Revit function, do not display the dictionary link
```

Comment:
I think the page has not yet been set up in the Dictionary - currently for Revit nodes, the link generated (e.g. http://dictionary.dynamobim.com/#/Revit/Analysis/FaceAnalysisDisplay/Create/ByViewAndFaceAnalysisData) takes the user to the Dictionary home page.